code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -34,6 +34,26 @@
 	///area this vent is assigned to
 	var/area/assigned_area
 
+	/// The integrity of the 'fan' for this vent; used to control the vent's ability to pump air.
+	var/fan_integrity = 1
+
+	/// Is this vent currently overclocked, removing pressure limits but damaging the fan?
+	var/fan_overclocked = FALSE
+
+	/// Rate of damage per atmos process to the fan when overclocked. Set to 0 to disable damage.
+	var/fan_damage_rate = 0.01
+
+	/// Datum for managing the overclock sound loop
+	var/datum/looping_sound/vent_pump_overclock/sound_loop
+
+/datum/looping_sound/vent_pump_overclock
```

Comment:
Put this with the other looping sound datums instead of in the middle of the vent pump file.